Launcher/Classes/ViewController.swift

```diff
@@ -684,5 +696,45 @@ class TasksViewController: NSViewController {
             strongSelf.createIRBSessionTask.waitUntilExit()
         }
     }
+    
+    func getCurrentDeviceUDID() -> String {
+        let simulatorUDIDs = "\(phoneComboBox.itemTitle(at: phoneComboBox.indexOfSelectedItem))"
+            .split(separators: "[]")
+            .map(String.init)
+        if simulatorUDIDs.count >= 2 {
+            return simulatorUDIDs[1]
+        }
+        return ""
+    }
+    
+    func changeLocale() {
+        var locale = "en"
+        var willRun = true
+        switch languagePopUpButton.title {
```

Comment:
If you make a `let titleLanguage = Language(rawValue: languagePopUpButton.title ?? "")`, you can switch on `.english`, `.german` etc. instead of using their `rawValue`. Right now, you could as well not use an enum.